README.md

```diff
@@ -1,3 +1,13 @@
 # Northcoders News API
 
-For instructions, please head over to [L2C NC News](https://l2c.northcoders.com/courses/be/nc-news).
+In order to use the test and development databases, please do the following
+In the root directory BE-BC-NEWS create two files
+
+.env.test which contains only:
+    PGDATABASE=nc_news_test
+
+.env.development which contains only
+    PGDATABASE=nc_news
+
+Next hit npm install
```

Comment:
consider wrapping any code or terminal commands in three backticks to create a code block or single backticks for inline code :)